packages/rs-platform-version/src/version/v2.rs

```diff
@@ -1247,7 +1247,7 @@ pub const PLATFORM_V2: PlatformVersion = PlatformVersion {
         },
         voting_versions: VotingVersions {
             default_vote_poll_time_duration_mainnet_ms: 1_209_600_000, //2 weeks
-            default_vote_poll_time_duration_test_network_ms: 5_400_000, //90 minutes
+            default_vote_poll_time_duration_test_network_ms: 1_209_600_000, //2 weeks
```

Comment:
You bring it back to keep compatibility with testnet data?

Reply:
yes